week2_assignment/hw2.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="kor">
+
```

Comment:
```suggestion
<html lang="ko">

```

html태그의 lang 속성을 작성할때는 kor이 아니라 ko라고 작성해주셔야합니다!